test/Feature/VersionSpec.hs

```diff
@@ -0,0 +1,25 @@
+module Feature.VersionSpec where
```

Comment:
I think this should be called `SchemaSpec` instead. It more accurately represents what the feature is vs the problem the feature aims to solve.